app/assets/javascripts/app/views/location_stream.js

```diff
@@ -21,10 +22,12 @@ app.views.LocationStream = app.views.Content.extend({
 
         L.marker(location).addTo(map);
         mapContainer.removeClass("empty");
+        footerContainer.classList.add("open");
         return map;
       }
     } else {
         mapContainer.toggle();
+        footerContainer.classList.remove("open");
```

Comment:
Meh, it seems to be a bug, doesn't it? I feel that the indentation is alright here.